backend/fms_core/models/project.py

```diff
@@ -12,7 +12,7 @@
 
 __all__ = ["Project"]
 
-STATUS_CHOICES = ["Ongoing", "Completed"]
+STATUS_CHOICES = ["Open", "Closed"]
```

Comment:
Little note here while i think about it. Maybe we should move this to _constants.py and rename it PROJECT_STATUS_CHOICES.